dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -0,0 +1,148 @@
+[[getting-traffic-into-cluster]]
+= Getting Traffic Into The Cluster
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+== Overview
+There are many ways to access the cluster. This section describes some
+commonly used approaches.
+
+The recommendation is:
+
+- If you have HTTP/HTTPS -- Use the xref:../dev_guide/getting_traffic_into_cluster.adoc#using-a-router[Router],
+- If you have Encrypted TCP (i.e., TLS with the SNI header) -- Use the xref:../dev_guide/getting_traffic_into_cluster.adoc#using-a-router[Router],
+- If you have TCP or UDP -- Use xref:../dev_guide/getting_traffic_into_cluster.adoc#using-the-loadbalancer[LoadBalancer],
+xref:../dev_guide/getting_traffic_into_cluster.adoc#using-externalIP[ExternalIP], or
+xref:../dev_guide/getting_traffic_into_cluster.adoc#using-nodeport[NodePort].
+
+The router is the most common way to access the cluster.
+This is limited to HTTP/HTTPS(SNI)/TLS(SNI) which covers web applications.
+
+TCP or UDP is useful when the HTTP protocol is not being used or non-standard
+ports are in use. There is more manual setup and monitoring involved.
+
+The admin is responsible for configuring the external network so that
+the DNS name lands on the exposed IP. When setting up a router it can be
+convenient to use wildcard DNS since this lets the users self-provision
+the router.
+
+The exposed IP addresses are commonly the IP addresses of nodes, Virtual
+IP addresses (VRRP) or Service NodePort addreses, Service externalIPs,
+Service ingressIP, etc..
```

Comment:
typo "adreses". Also, again, if you don't already know the thing that this is trying to tell you, then I don't think you'd be able to figure out what it's trying to tell you. (Eg, if you know what an ExternalIP is, then you don't need to be told anything about it here, but if you don't, this doesn't actually tell you anything about it, or even link to where you could learn more.)